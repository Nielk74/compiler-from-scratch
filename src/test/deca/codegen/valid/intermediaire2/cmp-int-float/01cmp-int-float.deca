// Description:
// Test de la conversion d'un int en float, lors d'une comparaison, lors de la condition d'un if.
// On test ici pour une égalité, qui retournera false.
//
// Resultats:
//  18 != 1.81000e+01
//
// Historique:
//    crée le 10/06/2022
{
  int x = 18;
  float y = 18.1;
  if (x == y) {
    println (x, " == " , y);
  } else {
    println (x , " != ", y);
  }
}
