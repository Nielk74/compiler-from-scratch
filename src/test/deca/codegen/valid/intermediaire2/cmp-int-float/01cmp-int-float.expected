18 != 1.81000e+01
